app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ]  @Yosefgeda great work adding associations in the comment model, however, you reference the user wrongly, instead, you should use the class and author eg `belongs_to :author, class_name: 'User'`